source/OctoVersion.Core/VersionNumberCalculation/SimpleVersion.cs

```diff
@@ -1,65 +1,36 @@
 ﻿using System;
-using System.Linq;
 
 namespace OctoVersion.Core.VersionNumberCalculation
 {
     public class SimpleVersion : IComparable
     {
-        public SimpleVersion(int major, int minor, int patch)
+        public SimpleVersion(int major, int minor, int? patch, int? build)
         {
             Major = major;
             Minor = minor;
-            Patch = patch;
+            Patch = patch.GetValueOrDefault();
+            Build = build.GetValueOrDefault();
         }
 
         public int Major { get; }
         public int Minor { get; }
         public int Patch { get; }
 
+        public int Build { get; }
```

Comment:
Why does this need to know about builds? It's calculated solely from the Git revision graph.